recipes/libtasn1/all/test_package/pkix.asn

```diff
@@ -0,0 +1,954 @@
+-- Copyright (C) 2002-2014 Free Software Foundation, Inc.
```

Comment:
Is it possible to reduce this file? Almost 1k lines, only for a simple test ...

Reply:
it's not source code, but a test file. I think it's okay (we e.g. have test jpeg files in repo)